src/components/widgets/EventWidget.tsx

```diff
@@ -71,7 +71,7 @@ export const EventWidget = ({ text }: WidgetProps) => {
 
   return (
     <DefaultWidget
-      count={eventCount}
+      count={additionalProps?.noCount ? null : eventCount}
```

Comment:
can we make the icon horizontally centered if the count is `null`? currently it looks a bit off 
<img width="63" alt="image" src="https://user-images.githubusercontent.com/1942953/188839939-4cf0d402-da51-4c6e-9d13-1f732d660d85.png">

Reply:
I thought about doing that. But if there is no number, we add a blank line. If there is no blank line, it looks like this. Which is better?
![Simulator Screen Shot - iPhone 13 - 2022-09-07 at 12 10 41](https://user-images.githubusercontent.com/11755668/188853165-67b1fc6a-b110-4782-8f9c-d3756f950365.png)

Reply:
this looks even more off. i meant that the icon gets horizontally centered above the text, you know?

Reply:
i think it looks better now. see the updated picture in the PR message.

Reply:
<img width="60" alt="image" src="https://user-images.githubusercontent.com/1942953/188902508-176c8941-dce8-4497-b463-f404771df954.png">

yes! that's it!